packages/components/src/components/telekom/telekom-header/telekom-header.tsx

```diff
@@ -32,6 +32,7 @@ export class TelekomHeader {
   @Prop({ reflect: true }) appName?: string;
   @Prop() appNameLink?: string;
   @Prop() appNameClick?: any;
+  @Prop() type: string = 'default';
```

Comment:
To keep it consistent this would have to be `standard`, no? 🤓

Reply:
```diff
- default
+ standard
```

it's only 1 character more haha…

But couldn't you make it `''` by default making the `type` prop optional?

Reply:
yes will do in a follow up PR